addon/services/import-rdfa-snippet.js

```diff
@@ -111,15 +120,22 @@ export default class ImportRdfaSnippet extends Service {
    * @method determineType
    */
   determineType(params, snippet, rdfaBlocks) {
-    const triples = rdfaBlocks.map((block) => block.context)
-          .reduce((prevValue, next) => [...prevValue,...next])
-          .uniq();
-    const types = triples.filter((triple) => triple.predicate === 'a').map((triple) => triple.object).uniq();
-    if (types.includes('https://data.vlaanderen.be/ns/mobiliteit#Verkeersbord-Verkeersteken')) {
-      return "roadsign";
-    }
-    else {
-      return "generic";
+    const triples = rdfaBlocks
```

Comment:
note that this is using marawa (that's ok), but this plugin doesn't have marawa as a dependency. It's quite likely we'll drop marawa from the editor at some point, so it would be best to add marawa as a dependency to this plugin